src/postmate.js

```diff
@@ -95,36 +101,30 @@ export class ParentAPI {
 
     this.events = {}
 
-    if (process.env.NODE_ENV !== 'production') {
-      log('Parent: Registering API')
-      log('Parent: Awaiting messages...')
-    }
+    log('Parent: Registering API')
+    log('Parent: Awaiting messages...')
 
     this.listener = (e) => {
       if (!sanitize(e, this.childOrigin)) return false
 
-      const { data, name } = e.data.value
+      const { data, name } = (((e || {}).data || {}).value || {})
```

Comment:
to jest teraz bez sensu - `sanitize` zapewnia ze dostajemy tutaj dobre dane juz, nie ma potrzeby na te `||` tuta - wystarczy uzyc `e.data.value`

Reply:
W testach wyszło że nie zawsze czego wynikiem jest acceptance test który nie przechodzi bez tych `||`